### DISCUSSION TITLE
<title>404 Githhub Page not found

### DISCUSSION THREAD

**Discussion by GLADDetailing** [community/community#58063] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I had recently connected my replit account to my existing GitHub repository. However, after testing out the new feature that allows you to make changes on replit and add them to the repository, my website was no longer publicly accessible. I keep getting a 404 page. Why?

**Comment by radar-blip** [comment]
If this is the one: https://gladdetailing.com/ it seems to be working now.

**Comment by GLADDetailing** [comment]
Yes it is thank you! I somehow resolved it earlier today.
